### PR TITLE
Update project with latest changes from iexec-core v8.5.0 and iexec-worker v8.5.0 releases

### PR DESCRIPTION
**Pull request by Natchica**

### CODE REVIEW

File: src/main/java/com/iexec/standalone/feign/CustomCoreFeignClient.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import com.iexec.common.config.WorkerModel;
 import com.iexec.common.replicate.ReplicateStatusUpdate;
 import com.iexec.standalone.api.SchedulerClient;
+import com.iexec.standalone.config.PublicConfiguration;
```
Comment (jbern0rd) [inline src/main/java/com/iexec/standalone/feign/CustomCoreFeignClient.java]: This import can be removed

File: src/test/java/com/iexec/standalone/configuration/ConfigServerClientConfigTests.java [hunk 2]
```diff
@@ -67,6 +67,7 @@ void checkChainConfigInitialization() {
                 .iexecHubContractAddress("0xC129e7917b7c7DeDfAa5Fff1FB18d5D7050fE8ca")
                 .blockTime(Duration.ofSeconds(5L))
                 .sidechain(true)
+                .iexecHubContractAddress("0xC129e7917b7c7DeDfAa5Fff1FB18d5D7050fE8ca")
```
Comment (jbern0rd) [inline src/test/java/com/iexec/standalone/configuration/ConfigServerClientConfigTests.java]: This line is already present above

File: src/test/java/com/iexec/standalone/replicate/ReplicateSupplyServiceTests.java [hunk 3]
```diff
@@ -1169,4 +1169,4 @@ private void mockWorkerCanAcceptMoreWork(final Worker worker) {
         when(workerService.getWorker(worker.getWalletAddress())).thenReturn(Optional.of(worker));
         when(workerService.canAcceptMoreWorks(worker)).thenReturn(true);
     }
-}
+}
```
Comment (jbern0rd) [inline src/test/java/com/iexec/standalone/replicate/ReplicateSupplyServiceTests.java]: I don't understand this change which adds an issue not present in scheduler code

Comment (Natchica) [reply src/test/java/com/iexec/standalone/replicate/ReplicateSupplyServiceTests.java]: Me neither ... it's probably due to the `rebase`.

File: CHANGELOG.md [hunk 4]
```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file.
 - Setup sms-related classes. (#15)
 - Setup result-related classes. (#16)
 - Setup compute- and tee-related classes. (#17)
+- Setup pubsub-related classes. (#18)
```
Comment (jbern0rd) [inline CHANGELOG.md]: We should find another wording and maybe replace the PR number with another one as it is no more by the PR under review.

Comment (jbern0rd) [reply CHANGELOG.md]:
```suggestion
- Setup pubsub-related classes. (#25)
```

**Review by jbern0rd — APPROVED** [review]

**Pull request merged**